deploy/charts/trust-manager/values.yaml

```diff
@@ -251,6 +251,16 @@ app:
 
         # The name of cert-manager's Service Account. Only used if `app.webhook.tls.approverPolicy.enabled` is true.
         certManagerServiceAccount: "cert-manager"
+
+      secretTemplate: {}
+        # Add labels/annotations to secrets created by Certificate resources
```

Comment:
I think the comments should be above the value, and not below. At least the description. Note that we generate the schema and docs.

Reply:
Alright, will move description of the key above and yes, I just noticed about schema and docs.

Reply:
You might have to try a bit locally and see that the schema and docs are generated correctly. :-)